plugins/pipelines/create.js

```diff
@@ -48,11 +48,17 @@ module.exports = () => ({
             const userFactory = request.server.app.userFactory;
             const scmUrl = formatScmUrl(request.payload.scmUrl);
             const username = request.auth.credentials.username;
+            let user;
 
             // fetch the user
             return userFactory.get({ username })
                 // get the user permissions for the repo
-                .then(user => user.getPermissions(scmUrl))
+                .then(u => {
+                    // store the user object for fetching token later
+                    user = u;
```

Comment:
I changed to use `user` object at [this line](https://github.com/screwdriver-cd/screwdriver/pull/194/files#diff-1eb6375f360812e82460d42f926123b1R82). I think it’s out of scope.